app/src/main/java/com/android/shelfLife/model/user/UserRepository.kt

```diff
@@ -62,16 +53,16 @@ interface UserRepository {
   suspend fun updateSelectedHousehold(selectedHouseholdUID: String)
 
   /** @param userEmails - The set of user emails to get the user IDs for. */
-  fun getUserIds(userEmails: Set<String?>, callback: (Map<String, String>) -> Unit)
+  suspend fun getUserIds(userEmails: Set<String?>): Map<String, String>
```

Comment:
were found out that repository function are not supposed to use callback functions, You have correctly updated our code